### PR TITLE
2.12.2

### PR DESCRIPTION
**Pull request by akravchenko564**

### CODE REVIEW

File: javascript.ru/2.12.2/2.12.2.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿var value = prompt('Are the unicorns real', '');
+
+if (value.toUpperCase() == 'Yes'.toUpperCase() ) {
```
Comment (KasichMax) [inline javascript.ru/2.12.2/2.12.2.js]: Minor: Redundant space after `'Yes'.toUpperCase()`.

File: javascript.ru/2.12.2/2.12.2.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+	alert('right!');
```
Comment (KasichMax) [inline javascript.ru/2.12.2/2.12.2.js]: `right` => `Right`.

File: javascript.ru/2.12.2/2.12.2.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+}
+
+else {
+	alert(' It is not right. Of course they are real!');
```
Comment (KasichMax) [inline javascript.ru/2.12.2/2.12.2.js]: `' It is not right...` =>`'It is not right...` (remove space).

File: javascript.ru/2.12.2/2.12.2.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿var value = prompt('Are the unicorns real', '');
```
Comment (KasichMax) [inline javascript.ru/2.12.2/2.12.2.js]: `the unicorns` => `unicorns` (because plural).

Comment (KasichMax) [inline javascript.ru/2.12.2/2.12.2.js]: Also where is question? MB you missed symbol `?`?

File: javascript.ru/2.12.2/2.12.2.js (hunk 3 above)
Comment (akravchenko564) [reply javascript.ru/2.12.2/2.12.2.js]: Done cbe8a98.

File: javascript.ru/2.12.2/2.12.2.js (hunk 2 above)
Comment (akravchenko564) [reply javascript.ru/2.12.2/2.12.2.js]: Done cbe8a98.

File: javascript.ru/2.12.2/2.12.2.js (hunk 1 above)
Comment (akravchenko564) [reply javascript.ru/2.12.2/2.12.2.js]: Done cbe8a98.

File: javascript.ru/2.12.2/2.12.2.js (hunk 4 above)
Comment (akravchenko564) [reply javascript.ru/2.12.2/2.12.2.js]: Done cbe8a98.

**Review by KasichMax — APPROVED** [review]

**Pull request merged**